src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="field_group">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+          class="field"
+          type="text"
+          name="user_surname"
+          autocomplete="off"
+          required
+          >
```

Comment:
![2023-01-13_20h08_15](https://user-images.githubusercontent.com/61904995/212389810-fb9d6ef8-5743-46b4-b0da-3e8053a12185.png)
Needs to fix in all places